tcs-persistence/src/main/resources/queries/traineeInfoForConnection.sql

```diff
@@ -17,6 +17,8 @@ from (
     -- note: null values are filtered out by the condition below
     and lower(gmc.gmcNumber) <> 'unknown'
     and gmc.gmcNumber not like CONCAT('%', UNHEX('c2a0'), '%') -- filter out all gmc number with non-breaking space
+  inner join Placement pl on (pl.id = cd.id )
```

Comment:
Requires change: `Placement.id` should be replaced with `Placement.traineeId` and the scope limited to "current" placements.

If not already considered: please describe if doctors without a current placement are intentionally excluded.

Reply:
Too slow :turtle: